src/badguy/badguy.cpp

```diff
@@ -170,11 +183,24 @@ BadGuy::draw(DrawingContext& context)
       }
       else
       {
-        if (m_frozen && is_portable())
+        if (m_frozen && is_portable()) {
           m_freezesprite->draw(context.color(), draw_pos, m_layer);
+        }
+
+        if (m_state != STATE_BURNING || m_firesprite->get_current_frame() < 5)
         m_sprite->draw(context.color(), draw_pos, m_layer - (m_frozen ? 1 : 0), m_flip);
```

Comment:
Please indent the body of this if statement
```suggestion
        if (m_state != STATE_BURNING || m_firesprite->get_current_frame() < 5)
          m_sprite->draw(context.color(), draw_pos, m_layer - (m_frozen ? 1 : 0), m_flip);
```